lib/browser.ts

```diff
@@ -668,6 +693,12 @@ export class ProtractorBrowser extends Webdriver {
       return 'Protractor.get(' + destination + ') - ' + str;
     };
 
+    if (this.bpClient) {
```

Comment:
Exactly. We could skip waiting for executeScript calls in general, but I wanted to keep behavior the same.